Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptDeleteController.java

```diff
@@ -141,23 +136,4 @@ public ModelAndView confirmlDelete(@PathVariable("id") String id,
         model.setViewName("redirect:/auth/" + listName + "/concepts");
         return model;
     }
-
-    @RequestMapping(value = "auth/conceptlist/deleteconcepts/{id}", method = RequestMethod.GET)
-    public ModelAndView deleteConcept(@PathVariable("id") String id,
-            @ModelAttribute("conceptSearchBean") ConceptSearchBean conceptSearchBean, BindingResult result,
-            Principal principal) throws LuceneException, IndexerRunningException {
-        ModelAndView model = new ModelAndView();
-        // Check if indexer is running
-        if (indexService.isIndexerRunning()) {
-            model.addObject("show_error_alert", true);
-            model.addObject("error_alert_msg", indexerRunning);
-            // Need to include command Object
-            model.setViewName("welcome");
-            return model;
-        }
-        conceptManager.deleteConcept(id, principal.getName());
-        model.setViewName("welome");
-        return model;
-    }
-
-}
```

Comment:
This method has been introduced due to merge conflicts issue. This method is not called from anywhere. I verified the request mapping.